Resources/doc/reference/configuration/match.rst

```diff
@@ -37,6 +38,14 @@ serve more than one host from your Symfony application.
 For example, ``path: ^/`` will match every request. To only match the home
 page, use ``path: ^/$``.
 
+``query_string``
+--------
+
+**type**: ``string``
+
+To match specific pages containing a parameter (let's say 'token' for instance),
```

Comment:
please say here that this is a regular expression. can you also make a second example to match a specific parameter value? `(^|&)token=value(&|$)` i would assume?

and please say something about url encoding to be clear whats the intention.